config.json

```diff
@@ -164,7 +164,7 @@
       "unlocked_by": null,
       "difficulty": 2,
       "topics": [
-        "dynamic_programming",
+        "dynamic-programming",
```

Comment:
I wonder if the reason for `jq` doing this differently has something to do with the website

Reply:
Hmm, I can't believe that the topics are actually supposed to be `snake_case`, but checking other repositories backs up that they are all in `snake_case`.

I guess I forgot, because it annoyed me so much :rofl:

Reply:
I guess `jq` just hates `snake_case` as much as I do, but I also hate `jq`, so ...

Anyway, it will be good for me to patch the functionality into the schene json library, so I'll keep working on it.